src/plugins/data/common/search/aggs/buckets/migrate_include_exclude_format.ts

```diff
@@ -18,6 +18,11 @@ export const isType = (...types: string[]) => {
   };
 };
 
+const isRegex = (text: string) => {
```

Comment:
ℹ️ I am using this function to identify that the value given is a pattern or a normal string.